tune_sklearn/_trainable.py

```diff
@@ -314,3 +320,102 @@ def reset_config(self, new_config):
         self.config = new_config
         self._setup(new_config)
         return True
+
+
+class _PipelineTrainable(_Trainable):
+    """Class to be passed in as the first argument of tune.run to train models.
+
+    Overrides Ray Tune's Trainable class to specify the setup, train, save,
+    and restore routines.
+
+    Runs all checks and configures parameters for the last step of the Pipeline.
+
+    """
+
+    def _can_partial_fit(self):
+        return check_partial_fit(self.main_estimator.steps[-1][1])
```

Comment:
what if you just overrode `self.main_estimator`?

Reply:
def main_estimator(self):
    return self.main_estimator.steps[-1][1]

Reply:
Done. Named it `base_estimator` to avoid conflicts